bower.json

```diff
@@ -22,7 +22,7 @@
     "angular-bootstrap-switch": "~0.5.1",
     "angular-dragdrop": "~1.0.13",
     "angular-mocks": "~1.5.8",
-    "angular-patternfly-sass": "~3.15.0",
+    "angular-patternfly-sass": "~3.21.0",
```

Comment:
It's [there](https://github.com/patternfly/angular-patternfly-sass/blob/28d795e4afe1c7245942c29bbec8b89c478bd652/bower.json#L42), I'm just not keeping the stuff in `master` :wink: